api.bs

```diff
@@ -142,8 +167,147 @@ The aggregation service:
 
 # API Details # {#api}
 
+Open questions:
+*   Filter/query language
+*   Reports are sent to aggregation system directly, or via conversion site? Or
+    option of either? => via conversion site
+*   Epochs
+
 TODO
 
+## ListAggregationSystems API ## {#list-aggregation-systems-api}
+
+navigator.privateAttribution.listAggregationSystems()
+
+<xmp class=idl>
+dictionary PrivateAttributionAggregationSystem {
+  required DOMString id;
+};
+</xmp>
+
+## SaveImpression API ## {#save-impression-api}
+
+<pre>
+navigator.privateAttribution.saveImpression({
+  type: "view",                  // either "view" or "click"
+  index: 3,                      // the histogram index for counting this impression
+  ad: "sample-campaign-eijb",    // a unique identifier for the ad placement
+  target: "advertiser.example",  // the advertiser site where a conversion will occur
+});
+</pre>
+
+Add:
+* attribution system
+* TTL
+* DP parameters
+
+Questions:
+*   Revisit the set of impression types. Can we get rid of it, and put it in the
+    ad ID? Or generalize to "attribution constraint"?
```

Comment:
Yes, let's remove that field.  The value here is explainability, but that's pretty marginal.  It's not like we expect regular users to be looking at what sites have stored.

It's also possible that we might gate click impressions on having a click, but I can't see how to use that information constructively.